Makefile

```diff
@@ -271,32 +271,63 @@ push-bundle-%:
 	\
 	rm -rf $${TMP_DIR}
 
+# Pushes just one bundle for the data
+# Once again use a tmp dir to prepare the content. This time it is
+# to make sure we avoid adding the data/config.json if it is there.
+#
+# The _ec_bundle.yml file is a way to avoid a problem where conftest
+# pushing the same image digest with a different tag causes an error.
+#
+.PHONY: push-data-bundle
+push-data-bundle: ## Create and push data bundle
+	@export \
+	  TMP_DIR="$$( mktemp -d -t ec-push.XXXXXXXXXX )" \
+	  TARGET="$(BUNDLE_REPO)/ec-policy-data:$(BUNDLE_TAG)" \
+	  DATA_FILES=$$( find $(DATA_DIR) -type f ! -name config.json ) && \
+	\
+	mkdir $${TMP_DIR}/$(DATA_DIR) && \
+	\
+	for f in $${DATA_FILES}; do \
+	  [[ -n $$( git status --porcelain $${f} ) ]] && \
+	    echo "Aborting due to uncommitted changes in $${f}!" && \
+	      exit 1; \
+	  cp $${f} $${TMP_DIR}/$(DATA_DIR); \
+	done && \
+	\
```

Comment:
Just noting that if we add the artifact hub manifest for this, the `find "$${TMP_DIR}" -name artifacthub-pkg.yml -delete && \` might be needed here